theme.json

```
@@ -1154,6 +1154,7 @@
 				}
 			},
 			"core/post-terms": {
+				"css": ".wp-block-post-terms a { white-space: nowrap; }",
```

Let's use the ampersand to avoid the duplicate selector.

```suggestion
				"css": "& a { white-space: nowrap; }",
```